src/ids.rs

```diff
@@ -121,7 +116,7 @@ mod test {
     #[test]
     fn test_large_indices() {
         let mut builder = IdTableBuilder::new();
-        builder.skip(1u64 << 33);
+        builder.next_id += 1u64 << 33;
```

Comment:
I have a feeling that we could also hide this bitshifts into function calls as beautification. But feel to merge if you don't want.

Reply:
I merge.